src/hotspot/share/gc/g1/g1ConcurrentMark.hpp

```diff
@@ -480,6 +480,8 @@ class G1ConcurrentMark : public CHeapObj<mtGC> {
   void clear_statistics(HeapRegion* r);
   // Notification for eagerly reclaimed regions to clean up.
   void humongous_object_eagerly_reclaimed(HeapRegion* r);
+  // Notification for old gen regions to clean up.
+  void old_region_reclaimed(HeapRegion* r);
```

Comment:
Why this decl? The rest look good.

Reply:
This is a split-off of a much larger change, and I overlooked this one.